src/EA8AE0.c

```diff
@@ -24,7 +24,28 @@ static char* N(exit_str_2) = "kzn_20";
 
 INCLUDE_ASM(s32, "EA8AE0", func_80240B00_EA95E0);
 
-INCLUDE_ASM(s32, "EA8AE0", func_80240BD4_EA96B4);
+typedef struct Unk80240BD4 {
+    f32 unk_0;
+    f32 unk_4;
+    f32 unk_8;
+} Unk80240BD4;
```

Comment:
Can you add `/* 0x00 */` etc comments? Also name the fields `unk_00` etc (2 digits)

Reply:
done